ckanext/opendatani/helpers.py

```diff
@@ -140,8 +138,9 @@ def is_admin(user, org):
 
     return any(
         [(i.get('capacity') == 'admin')
-         and i.get('name') == org for i in user_orgs])\
-        or authz.is_sysadmin(user)
+         and i.get('name') == org for i in user_orgs]) \
+        or authz.is_sysadmin(user) or (authz.is_sysadmin(user)
+                                       and org == 'full-portal')
```

Comment:
Is no this redundant? if `sysadmin` it should return True no matter what is `org` is

Reply:
Good catch... I don't know how I missed that.